tql_macros/src/gen/mod.rs

```diff
@@ -28,7 +28,7 @@ mod sqlite;
 
 use proc_macro::TokenStream;
 use proc_macro2::Span;
-use quote::Tokens;
+use proc_macro2::TokenStream as Tokens; // todo: remove
```

Comment:
Is this comment this useful?

Reply:
not really, at the beggining i was just thinking about removing it / renaming `Tokens` to `TokenStream2` to keep naming more consistent. I will remove this comment for now